generator/.DevConfigs/feature-net8.json

```
@@ -1,14 +1,18 @@
 {
   "core": {
-    "changeLogMessages": ["Add trimming support for .NET 8."],
-    "type": "patch",
-    "updateMinimum": true,
+    "changeLogMessages": [
+		"Add .NET 8 framework target for core and service projects.",
+		"For .NET 8 mark all SDK assemblies as trimmable and address all trim warnings.",
+		"For .NET 8 prevent the System.Uri class from canonicalizing the resource path. This broke scenarios with S3 object keys like `foo/../bar.txt`.",
+		"For .NET 8 use the System.Net.Http.HttpClient synchronous `Send` method when then internal synchronous service request methods are invoked."
```

I don't follow the last changelog message, did you mean:
```suggestion
		"For .NET 8 use the System.Net.Http.HttpClient synchronous `Send` method when their internal synchronous service request methods are invoked."
```

---

I am thinking `then` should be deleted and replaced with nothing.